content/terraform-plugin-testing/v1.13.x/docs/plugin/testing/acceptance-tests/teststep/import-mode.mdx

```diff
@@ -0,0 +1,382 @@
+---
```

Comment:
TODO: move file from `teststep/import-mode.mdx` to `import-mode.mdx`. This fixes the link in the nav.